tests/Handler/Mapping/ClassName/SuffixTest.php

```diff
@@ -12,12 +12,19 @@ class SuffixTest extends TestCase
     /** @dataProvider examples */
     public function testCanAddSuffixToClassName(string $suffix, string $command, string $expectedResult) : void
     {
-        self::assertEquals(
+        self::assertSame(
             $expectedResult,
             (new Suffix($suffix))->getClassName($command)
         );
     }
 
+    public function testBuildHandlerSuffix(): void
+    {
+        $result = Suffix::handler()->getClassName('MyCommand');
+
+        self::assertSame('MyCommandHandler', $result);
+    }
+
```

Comment:
In line with previous comment about ::handle this would go too, sorry :crying_cat_face: